app/src/main/java/com/example/user/epam_hw/FragmentDownloads.java

```diff
@@ -0,0 +1,38 @@
+package com.example.user.epam_hw;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+public class FragmentDownloads extends Fragment {
+
+    public static final String ARG_PAGE = "ARG_PAGE";
+
+    private int mPage;
+
+    public static FragmentDownloads newInstance(int page) {
+        Bundle args = new Bundle();
+        args.putInt(ARG_PAGE, page);
+        FragmentDownloads fragment = new FragmentDownloads();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            mPage = getArguments().getInt(ARG_PAGE);
+        }
+    }
+
+    @Override public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                                       Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.downloads_fragment, container, false);
```

Comment:
Class name should be SomeNameTypeOfFrameworkClass: DownloadFragment but layouts has this type: type_of_framework_class_name_of_class - fragment_download, activity_main, view_profile etc